packages/db/queries/health.sql

```diff
@@ -0,0 +1,2 @@
+-- name: Health :one
```

Comment:
Simple query that pings the database and verifies the pool of connections is actually ready.